compiler/luci-interpreter/src/kernels/TransposeConv.test.cpp

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "kernels/TransposeConv.h"
+#include "kernels/TestUtils.h"
+
+namespace luci_interpreter
+{
+namespace kernels
+{
+namespace
+{
+
+using namespace testing;
+
+template <typename T>
+void Check(std::initializer_list<int32_t> outputShape_shape,
+           std::initializer_list<int32_t> weight_shape,
+           std::initializer_list<int32_t> inputData_shape,
+           std::initializer_list<int32_t> output_shape, std::initializer_list<T> outputShape_data,
+           std::initializer_list<T> weight_data, std::initializer_list<T> inputData_data,
+           std::initializer_list<T> output_data, luci::Padding padding, int32_t stride_height,
+           int32_t stride_width, DataType element_type)
+{
+  Tensor outputShape_tensor{element_type, outputShape_shape, {}, ""};
```

Comment:
Please use snake_case for variable names.

Reply:
Okay, I will fix it.